excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/CategoryParser.java

```diff
@@ -71,6 +71,8 @@ public CategoryEntity createCategoriesEntity(ParseContext parseContext,
         categoriesEntity.setParentCategoryId(definitionDataItem.getString(ColumnName.PARENT_CATEGORY_ID));
         categoriesEntity.setCategoryLabel(definitionDataItem.getString(ColumnName.CATEGORY_LABEL));
         categoriesEntity.setCategoryId(definitionDataItem.getString(ColumnName.CATEGORY_ID));
+        //register Case Type and category
+        parseContext.registerCaseTypeForCategory(caseType, categoriesEntity);
```

Comment:
This is mutating an input parameter, `parseContext` in this case.  I think this code belongs somewhere else.